### PR TITLE
Add cdk

### PR DESCRIPTION
**Pull request by tomrf1**
The first step in using cdk is to add it to the project, and just point it at our existing cloudformation.yaml.
Later, we can redefine our resources in TS using the cdk api, and remove the cloudformation.yaml.

### CODE REVIEW

File: cloudformation.yaml [hunk 1]
```diff
@@ -203,8 +203,6 @@ Resources:
       Name: !Sub ${Stack}-${Stage}-${App}
       Port: 9000
       Protocol: HTTP
-      VpcId:
-        Ref: VpcId
```
Comment (tomrf1) [inline cloudformation.yaml:207]: this is the only change. Here `VpcId` appeared twice (by mistake), which now produces an error:
`Map keys must be unique; "VpcId" is repeated`

File: jest.config.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+module.exports = {
```
Comment (tomrf1) [inline jest.config.js:1]: the jest config in package.json wasn't working properly, I've moved it here and fixed the path condition

File: .eslintignore [hunk 3]
```diff
@@ -0,0 +1,12 @@
+cdk/
```
Comment (tomrf1) [inline .eslintignore:1]: eslint was previously using the .gitignore file, but we now need to exclude the cdk directory from the main linter, so I've added a .eslintignore

File: cloudformation.yaml (hunk 1 above)
Comment (TomPretty) [reply cloudformation.yaml:207]: good to fix this!

**Review by TomPretty — COMMENTED** [review]
Looking good! I think we'll need to update https://github.com/guardian/support-admin-console/blob/main/build.sbt#L41 to use the new cloudformation file produced by cdk. Given it produces json it might be worth renaming to `cfn.json` and also updating riff-raff: https://github.com/guardian/support-admin-console/blob/main/conf/riff-raff.yaml#L8

**Comment by tomrf1** [comment]
> Looking good! I think we'll need to update https://github.com/guardian/support-admin-console/blob/main/build.sbt#L41 to use the new cloudformation file produced by cdk. Given it produces json it might be worth renaming to `cfn.json` and also updating riff-raff: https://github.com/guardian/support-admin-console/blob/main/conf/riff-raff.yaml#L8

Good spot, thanks!

**Review by TomPretty — APPROVED** [review]
LGTM! Thanks for picking this one up in marketing tools!

**Pull request merged**